src/Tizen.NUI/src/public/BaseComponents/TextFieldEvent.cs

```diff
@@ -208,6 +246,20 @@ private void OnTextChanged(IntPtr textField)
             }
         }
 
+        private void OnCursorPositionChanged(IntPtr textField, uint oldPosition)
+        {
+            if (textFieldCursorPositionChangedEventHandler != null)
+            {
+                CursorPositionChangedEventArgs e = new CursorPositionChangedEventArgs();
+
+                // Populate all members of "e" (CursorPositionChangedEventArgs) with real data
+                e.CursorPosition = oldPosition;
```

Comment:
The name is CursorPosition, but it is actually OldCursorPosition.
This can be confusing to users.
I think this should be "OldCursorPosition".

But you don't have to provide it now, As you know, now we dont't have time.
We have already implemented this in DALi, so you can add it later as needed.

Reply:
So you mean for now this will be without event args ? remove CursorPositionChangedEventArgs class?

Reply:
I think OldPosition can be implemented easily in the app.
However, we have always been criticized for poor usability.
So I think this may be provided.
What do you think? I think this can be decided by us.
If you remove CursorPositionChangedEventArgs , you should also remove SelectionChangedEventArgs, for consistency.

Reply:
I think it is better to keep old position as event args, it is more convenient for the user if he is interested in the old value.
for the naming, if I use OldCursorPosition doesn't this imply that there is a NewCursorPosition ?

Reply:
```
I think it is better to keep old position as event args, it is more convenient for the user if he is interested in the old value.
```
Ok, I agree with you.

```
for the naming, if I use OldCursorPosition doesn't this imply that there is a NewCursorPosition ?
```
However, if there is only CursorPosition in the event args, it seems to cause confusion for users. Isn't it?

Reply:
yes it could, but reading the docs should clear things.

Reply:
anyway I have changed it to Old